packages/components/src/components/Radio/RadioGroup.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {
+  Root as RadioGroupPrimitiveRoot,
+  RadioGroupProps as RadioGroupPrimitiveProps,
+} from "@radix-ui/react-radio-group";
+import { Box } from "../../primitives/Box";
+
+export interface RadioGroupProps extends RadioGroupPrimitiveProps {
+  /** The individual Radio components within the group. */
+  children: React.ReactNode;
+  /** The value of the radio item that should be checked when initially rendered. Use when you do not need to control the state of the radio items. */
+  defaultValue?: string;
+  /** The reading direction of the radio group. If omitted, assumes LTR (left-to-right) reading mode. */
+  dir?: "ltr" | "rtl";
```

Comment:
Do we need this for now? 🤔

Reply:
It's fine to leave it in place for now. We'll need to start thinking about this eventually with our components.